### PR TITLE
chemistry and catalog fixes

### PR DESCRIPTION
**Pull request by ITCups**
## About The Pull Request
<details>
<summary>
	Fixes and new tablet for chemist that someone should map in
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

**Comment by SpadesSlick** [comment]
Wait, can't we just add this to their tablet they spawn with?

**Comment by ACCount12** [comment]
That makes sense actually. Thought it was already there.

**Comment by SpadesSlick** [comment]
It was, like, whole purpose of having it as modular program.

**Comment by SpadesSlick** [comment]
Add it for all of the Moebius personal, no need to hide basically in-game wiki,

**Comment by ACCount12** [comment]
Why is it tablet only in the first place?

File: code/game/machinery/machinery.dm [hunk 1]
```diff
@@ -245,13 +245,13 @@ Class Procs:
...
 	if(!istype(type,/obj/item/weapon/stock_parts))
-		return
+		return 1
 	var/list/obj/item/weapon/stock_parts/parts = list()
 	for(var/list/obj/item/weapon/stock_parts/P in component_parts)
 		if(istype(P, type))
 			parts.Add(P)
 	if(!parts.len)
-		return
+		return 1
```
Comment (LethalGhost) [inline code/game/machinery/machinery.dm]: Why `error case` return `1` instead `null` as it was described in comment near proc head?

**Pull request merged**